<html>
<head><meta charset="utf-8"><title>merge permissions on wasmtime-py? · wasmtime · Zulip Chat Archive</title></head>
<h2>Stream: <a href="https://bytecodealliance.github.io/zulip-archive/stream/217126-wasmtime/index.html">wasmtime</a></h2>
<h3>Topic: <a href="https://bytecodealliance.github.io/zulip-archive/stream/217126-wasmtime/topic/merge.20permissions.20on.20wasmtime-py.3F.html">merge permissions on wasmtime-py?</a></h3>

<hr>

<base href="https://bytecodealliance.zulipchat.com">

<head><link href="https://bytecodealliance.github.io/zulip-archive/style.css" rel="stylesheet"></head>

<a name="210775463"></a>
<h4><a href="https://bytecodealliance.zulipchat.com#narrow/stream/217126-wasmtime/topic/merge%20permissions%20on%20wasmtime-py%3F/near/210775463" class="zl"><img src="https://bytecodealliance.github.io/zulip-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> fitzgen (he/him) <a href="https://bytecodealliance.github.io/zulip-archive/stream/217126-wasmtime/topic/merge.20permissions.20on.20wasmtime-py.3F.html#210775463">(Sep 21 2020 at 17:44)</a>:</h4>
<p>Apparently I don't have merge permissions for <code>wasmtime-py</code>, but this PR LGTM: <a href="https://github.com/bytecodealliance/wasmtime-py/pull/45">https://github.com/bytecodealliance/wasmtime-py/pull/45</a></p>
<div class="message_embed"><a class="message_embed_image" href="https://github.com/bytecodealliance/wasmtime-py/pull/45" style="background-image: url(https://avatars0.githubusercontent.com/u/54038801?s=400&amp;v=4)"></a><div class="data-container"><div class="message_embed_title"><a href="https://github.com/bytecodealliance/wasmtime-py/pull/45" title="Fixed Python hello example on README.md by viniarck · Pull Request #45 · bytecodealliance/wasmtime-py">Fixed Python hello example on README.md by viniarck · Pull Request #45 · bytecodealliance/wasmtime-py</a></div><div class="message_embed_description">When I first tried out the example in the README.md, I hit this TypeError exception:
Traceback (most recent call last):
  File &quot;ex.py&quot;, line 4, in &lt;module&gt;
    module = Module(store...</div></div></div>



<a name="210775500"></a>
<h4><a href="https://bytecodealliance.zulipchat.com#narrow/stream/217126-wasmtime/topic/merge%20permissions%20on%20wasmtime-py%3F/near/210775500" class="zl"><img src="https://bytecodealliance.github.io/zulip-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> fitzgen (he/him) <a href="https://bytecodealliance.github.io/zulip-archive/stream/217126-wasmtime/topic/merge.20permissions.20on.20wasmtime-py.3F.html#210775500">(Sep 21 2020 at 17:44)</a>:</h4>
<p>so it would be nice if someone who did have perms merged it</p>



<a name="210903596"></a>
<h4><a href="https://bytecodealliance.zulipchat.com#narrow/stream/217126-wasmtime/topic/merge%20permissions%20on%20wasmtime-py%3F/near/210903596" class="zl"><img src="https://bytecodealliance.github.io/zulip-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Till Schneidereit <a href="https://bytecodealliance.github.io/zulip-archive/stream/217126-wasmtime/topic/merge.20permissions.20on.20wasmtime-py.3F.html#210903596">(Sep 22 2020 at 17:23)</a>:</h4>
<p>I see Alex merged the PR, but I now also gave wasmtime-core write permissions to the repo. One of these days we'll clean all of this up ...</p>



<hr><p>Last updated: Jan 20 2025 at 06:04 UTC</p>
</html>